apps/web/src/components/equipment/IpAddressInput.tsx

```
@@ -0,0 +1,293 @@
+ */
+
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import {
+  Box,
+  CircularProgress,
+  IconButton,
+  InputAdornment,
+  TextField,
+  Tooltip,
+} from '@mui/material';
+import { Check, ContentCopy, Error, Router, Warning } from '@mui/icons-material';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)